plugins/unraid/.claude-plugin/README.md

```
@@ -2,34 +2,38 @@
 ---
 
 ## Tools
 
-### `unraid` — Primary Tool (108 subactions, 15 domains)
+### `unraid` — Primary Tool
 
 Call as `unraid(action="<domain>", subaction="<operation>", [params])`.
+For the complete current action/subaction reference, call `unraid(action="help")`.
 
-#### `system` — Server Information (18 subactions)
+#### `system` — Server Information
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)